dm/syncer/ddl_test.go

```diff
@@ -670,9 +697,13 @@ func (s *testDDLSuite) TestAdjustCollation(c *C) {
 		Schema: "test",
 		Name:   "t",
 	}
-	statusVars := []byte{4, 0, 0, 0, 0, 46, 0}
+	statusVars := []byte{4, 0, 0, 0, 0, 21, 1}
 	charsetAndDefaultCollationMap := map[string]string{"utf8mb4": "utf8mb4_general_ci"}
+	idAndCollationMap := map[int]string{46: "utf8mb4_bin", 277: "utf8mb4_vi_0900_ai_ci"}
 	for i, sql := range sqls {
+		if i == 1 {
```

Comment:
you can test it in another testing function. Two same "create database if not exists `test`" is very misleading

Reply:
ok